remarkable-review/The End of Paper.md

```diff
@@ -0,0 +1,101 @@
+
+<photo of Eric with a "paper notebook">
+
+I wrote nearly every day. I also evolved my system continually. I moved from a single notebook for capturing everything to two: one for a daily working log and one for personal, long-form journaling. I experimented with different formatting techniques for pages. I dabbled in a variety of paper todo systems. I experimented with different notebook form factors: soft cover, hard cover, small, medium, large, lined, unlined, grid. I scribbled my way through sheafs of dead tree matter. Writing even seem to reshape my brain: my habit of thought defaulted to externalizing ideas on paper or whiteboard.
+
+And I loved it.
+
+I loved starting a new day with a fresh page, noting the day's date, and opening myself to the possibilities in front of me. I loved dedicating time to look back on life, sorting through emotions, churning thought-trains, and half-made plans. From brain to hand to pen, the ink flowing onto the page created a black-and-beige clarity that seemed more difficult when staring at a glowing rectangle or typing on my keyboard.
+
+All that's to say, I was and am still deeply in love with paper. 
+
+## A dalliance with digital
+As wonderful as paper had been to me, as a digital native, I was constantly aware of the tradeoff I was making: digital has its advantages.
+
+I got my first job as a software developer in 2003. Which means I type a lot. My keyboard is often the most expedient tool for making quick notes.
+
+It began with simple "scratch notes", outlining how I was planning to tackle a problem or something I wanted to remember for later in the day. Eventually, I found myself needing to recall something from a previous day or share something with a colleague.
+
+In 2011, I decided my digital note-taking needed a system of it's own. I signed up for [Simplenote](https://simplenote.com/) and began accumulating notes. Over the next decade, I migrated to NVAlt and then Bear, bringing my growing collection of notes with me each time. In tandem with my paper note-taking practice, I iterated on my digital note-taking process.
+
+Today, I have 706 notes, including this article. I've been bitten by Zettelkasten bug. If you haven't heard of that particular insect, it's mostly harmless: it causes you to lose track of time whenever you have an idea.
+
+If you're really into organizational systems, I recorded a [livestream](livestream) where I whiteboarded my reading, writing, and note-taking systems.
+
+My digital note repository does have advantages over my multitudinous Moleskines.
```

Comment:
This feels like the start of a new section - the analog vs digital part of our story.